src/wrapper/util.rs

```diff
@@ -28,3 +31,11 @@ pub fn find_cmd(cmd: &str, env: &BTreeMap<String, String>)
         }
     }
 }
+
+pub fn check_data_container(container_config: &Container) {
```

Comment:
`check_data_container` -> `warn_if_data_container`

Current name implies some check if data container has appropriate settings, rather than what it does